click-to-reveal/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Click to reveal</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="./script.js"></script>
+</head>
+
+<body>
+
+    <section id="idt-click-to-reveal">
+        <h1>Click to reveal</h1>
+        <div>
+
+            <div id="idt-btn-container">
+
+            </div>
+            <div id="container">
```

Comment:
`container` is too generic for an id and will likely collide with other IDs.

```suggestion
            <div id="idt-container">
```